### PR TITLE
Move out cli and add autocomplete support

### PR DESCRIPTION
**Pull request by romasku**
Closes https://github.com/neuromation/neuro-flow/issues/32

### CODE REVIEW

**Comment by romasku** [comment]
I think we will need to separate runners logic from printing code in (possibly far) future. The current situation makes the autocomplete code to know a lot about how runners work.

**Comment by romasku** [comment]
I have moved a lot of code, I you think it was bad idea - I will try to return it back to `cli.py` file.

File: neuro_flow/cli/completion.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            content = profile.read()
+    except FileNotFoundError:
+        content = b""
+
+    start = content.find(GENERATED_START)
+    if start != -1:
+        end = content.find(GENERATED_END)
+        if end == -1:
+            raise click.ClickException(
+                f"Malformed guarding comments. Please modify {profile_file} manually"
```
Comment (asvetlov) [inline neuro_flow/cli/completion.py:68]:
```suggestion
                f"Malformed guarding comments. Please edit {profile_file} manually"
```

**Review by asvetlov — APPROVED** [review]
LGTM

**Pull request merged**